### ISSUE TITLE
VBox + Hyper-V.  Installer makes page fault in _WUTL951.DLL.  Possible work around?

### ISSUE THREAD

**Issue by fransking** [JHRobotics/softgpu#83]
I ran in the problem you described [here](https://www.vogons.org/viewtopic.php?f=24&t=88284&start=40) when using VirtualBox with Hyper-V.

I noticed that setting the OS type in VirtualBox to Windows 98 instead of Linux 32bit makes this problem go away but I was then presented with an invalid settings warning and the display adapter is set back to VBoxVGA.  

Neverthless, you can still set it to VBoxSVGA on the VM summary window and it seems to be working as of version 7.0.18.  GL Checker reports GL Version 3.3 and render as SVGA3D VMware.

Is anyone else able to test and confirm?

**Comment by songday** [comment]
Not sure whether you've read this thread: https://www.vogons.org/viewtopic.php?p=1194719  
According to
```
And finally, another possible bug: if VirtualBox is not running natively but under WHPX (Microsoft's Hyper-V and the Windows Hypervisor Platform). Beside incredibly slow I/O performance makes this error: Near half of games can’t be installed because installer makes page fault in _WUTL951.DLL. 
```
you should turn off `Hyper-V`

**Comment by fransking** [comment]
Yes I had read this but I was looking (and may have found) a solution which doesn’t require turning off Hyper-V which I want to keep for things like WSL2.On 16 May 2024, at 05:35, Songday ***@***.***> wrote:﻿
Not sure whether you've read this thread: https://www.vogons.org/viewtopic.php?p=1194719
According to
And finally, another possible bug: if VirtualBox is not running natively but under WHPX (Microsoft's Hyper-V and the Windows Hypervisor Platform). Beside incredibly slow I/O performance makes this error: Near half of games can’t be installed because installer makes page fault in _WUTL951.DLL. 

you should turn off Hyper-V

**Comment by steviator** [comment]
Sadly Microsoft are doing a pretty good job of sabotaging third-party VMs on the Windows desktop, probably to mitigate a bunch of historic security blunders without actually needing to fix anything.  

Incidentally, they're also sabotaging their own VM on servers to try to force corporate users into the cloud. 

It must be very complicated being Microsoft these days.

**Comment by mbaran5** [comment]
@fransking I am unable to replicate, are you saying that setting the machine back to Windows 98 resolves the WUTL951.dll error? When I tried that, I was still presented with the same error installing software.
